schema/src/main/scala/CpgExtSchema.scala

```diff
@@ -228,6 +228,11 @@ class CpgExtSchema(builder: SchemaBuilder, cpgSchema: CpgSchema) {
   private val derivedSourceEdge  = builder.addEdgeType(CpgSchemaConstants.DERIVED_SOURCE_EDGE_NAME)
   astNode.addOutEdge(edge = originalSourceEdge, inNode = astNode)
   astNode.addOutEdge(edge = derivedSourceEdge, inNode = astNode)
+
+  // Extend Dependency Node
+  dependency.addOutEdge(edge = sourceFile, inNode = file)
+  dependency.addOutEdge(edge = taggedBy, inNode = tag)
+  dependency.extendz(astNode)
```

Comment:
Place these three lines with line number 165, so all dependency related information is clubbed

Reply:
done